R/pmcmc.R

```diff
@@ -640,9 +637,8 @@ summary.pmcmc <- function(object, ...) {
   
   sds <- round(apply(traces, 2, sd), 3)
   # convert start_date back into dates
-  summ$start_date <- offset_to_start_date(data_start_date, summ$start_date)
-  summ$start_date <- as.Date(summ$start_date, origin="2019-12-31")
-  summ[c('2.5%', '97.5%', 'min', 'max'), 'start_date'] <- summ[c('97.5%', '2.5%', 'max', 'min'), 'start_date']
```

Comment:
Did you want to remove this line?

Reply:
Yes, when we used the offsets the CrI on start_date would appear the wrong way round when we converted back to dates so this was just housekeeping to deal with that. Now that we use sircovid_dates the 2.5% and 97.5% CrIs no longer need to be switched